features/routing/route.feature

```diff
@@ -80,10 +80,13 @@ Feature: Testing route
     When I run the :create client command with:
```

Comment:
@quarterpin, not related to this PR, but can you please use the step `Given I obtain...` to get the testdata?  You can refer #1227 for examples on how to do this.

Reply:
Hello @pruan-rht, 
As per your suggestions, I have modified the case to utilize the `I obtain` statement. Successful v3-runner job post the change is as below: 
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/103425/console

Let me know if there is anything further to change/modify. 

Thanks